### PR TITLE
Contribute to James howto

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by MichaelBailly** [comment]
:+1:

File: _posts/2016-2-6-contribute-to-mail-server.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tip-tldr: Contributing to an awesome mail server.
+---
+
+OpenPaas is contributing to a mail server of the [Apache Software Foundation](http://www.apache.org/), which is called [James](http://james.apache.org/).
+
+Not as others OpenPaas technologies, this component is a Java base project.
+
+One of our main goal is to provide a cloud ready mail server. In order to achieve that, we are using technologies such as Cassandra, ElasticSearch, Docker and much more.
+
+As OpenPaas is web-oriented, the legacy mail protocol (IMAP) doesn't feet our needs, so we are also implementing a REST mail protocol which is called [JMAP](http://jmap.io/).
```
Comment (GrahamLinagora) [inline _posts/2016-2-6-contribute-to-mail-server.md]: IMO `does not fit` instead of `doesn't feet`

Comment (aduprat) [inline _posts/2016-2-6-contribute-to-mail-server.md]: OK

**Comment by GrahamLinagora** [comment]
:+1:

**Pull request merged**